src/page-multi-facility/ReferenceDataModal/SyncNoUserFacilities.tsx

```diff
@@ -282,6 +310,21 @@ const SyncNoUserFacilities: React.FC = () => {
   const [modalTitle, setModalTitle] = useState<string>();
   const [systemType, setSystemType] = useState<string>();
   const [stateName, setStateName] = useState<string>();
+  const [scenarioState, dispatchScenarioUpdate] = useScenario();
+  const scenario = scenarioState.data;
+  const scenarioId = scenario?.id;
+
+  // Immediately set the referenceDataObservedAt time so that
+  // this modal does not compete with the SyncNewReferenceData
+  // modal for attention.
+  useEffect(() => {
+    saveScenario({
+      ...scenario,
+      referenceDataObservedAt: new Date(),
+    }).then((savedScenario) => {
+      if (savedScenario) dispatchScenarioUpdate(savedScenario);
+    });
+  }, [scenarioId]);
```

Comment:
if you intended to exclude the missing dependencies here, please disable this lint rule and add some kind of explanatory comment

Reply:
Yes, thanks for pointing this out.  Turns out I actually didn't need any dependencies so it was worth taking a second look.

Reply:
Is there a way to get these checks to run locally?  The pre-commit hook catches some stuff, but not everything.

Reply:
you can run `yarn lint` or install an ESLint integration in your editor if there is one ... these are currently just warnings rather than errors though, which is why the pre-commit hook doesn't fail on them. (Also, we could change it so that they are errors, since this seems to come up a lot in pull requests.)